### PR TITLE
Gl 862 

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/GL-862

### What?
    - Various UR2 design improvement
    - Add the classification to the "Your goods" section in the result page.
    - Refactored logic into the classifiable concern.

### Note
this is part of the GL-862,
more changes on it are coming.

### CODE REVIEW

File: app/models/concerns/classifiable.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module Classifiable
```
Comment (shjohnson) [inline app/models/concerns/classifiable.rb:1]: Is this tested anywhere like in another class as side affect?

File: app/views/green_lanes/results/_result_category_1.html.erb [hunk 2]
```diff
@@ -1,36 +1,54 @@
 <div class="govuk-panel govuk-panel--confirmation">
   <h1 class="govuk-panel__title">
-    Your goods are Category 1
```
Comment (shjohnson) [inline app/views/green_lanes/results/_result_category_1.html.erb:3]: Should this be changed?

File: app/views/green_lanes/shared/_summary_card.html.erb [hunk 3]
```diff
@@ -11,6 +11,12 @@
           <dt class="govuk-summary-list__key">Commodity</dt>
           <dd class="govuk-summary-list__value"><%= @commodity_code %></dd>
         </div>
+        <div class="govuk-summary-list__row">
```
Comment (shjohnson) [inline app/views/green_lanes/shared/_summary_card.html.erb:14]: 👏

**Review by shjohnson — COMMENTED** [review]
Happy to approve but there is one comment about testing on the classification class

File: app/models/concerns/classifiable.rb (hunk 1 above)
Comment (alexdesi) [reply app/models/concerns/classifiable.rb:1]: Yes,  good point. This is tested in:
`spec/models/commodity_spec.rb`

Comment (willfish) [reply app/models/concerns/classifiable.rb:1]: And as it gets rendered in the views

File: app/models/green_lanes/goods_nomenclature.rb [hunk 4]
```diff
@@ -13,9 +14,7 @@ class GreenLanes::GoodsNomenclature
 
   has_many :applicable_category_assessments, class_name: 'GreenLanes::CategoryAssessment'
 
-  def all(_opts = {})
```
Comment (willfish) [inline app/models/green_lanes/goods_nomenclature.rb:16]: This makes me so happy

**Review by willfish — APPROVED** [review]

**Pull request merged**